doc/user_manual/rom.tex

```diff
@@ -2753,99 +2755,114 @@ \subsubsection{SciKitLearn}
+ % autocorrelation function returning the autocorrelation between two points $x$
+ % and $x'$.
+ % %
+ % Default assumes a squared-exponential autocorrelation model.
+  %
+  %Built-in correlation models are `absolute\_exponential,'
+  %`squared\_exponential,' `generalized\_exponential,' `cubic,' and `linear.'
+  %
+ % \default{squared\_exponential}
+%  \item \xmlNode{beta0}, \xmlDesc{float, array-like, optional field}, specifies
+%  the regression weight vector to perform Ordinary Kriging (OK).
+%  %
+%  \default{None}
+%  \item \xmlNode{storage\_mode}, \xmlDesc{string, optional field}, specifies
+%  whether the Cholesky decomposition of the correlation matrix should be stored
+%  in the class (storage\_mode = `full') or not (storage\_mode = `light').
+%  %
+%  \default{full}
+%  \item \verDescriptionA{False}
+%  \item \xmlNode{theta0}, \xmlDesc{float, array-like, optional field}, is an
+%  array with shape (n\_features, ) or (1, ).
+%  %
+%  This represents the parameters in the autocorrelation model.
+%  %
+%  If thetaL and thetaU are also specified, theta0 is considered as the starting
+%  point for the maximum likelihood estimation of the best set of parameters.
+%  %
+%  \default{[1e-1]}
+%  \item \xmlNode{thetaL}, \xmlDesc{float, array-like, optional field}, is an
+%  array with shape matching that defined by \xmlNode{theta0}.
+%  %
+%  Lower bound on the autocorrelation parameters for maximum likelihood
+%  estimation.
+%  %
+%  \default{None}
+%  \item \xmlNode{thetaU}, \xmlDesc{float, array-like, optional field}, is an
+%  array with shape matching that defined by \xmlNode{theta0}.
+%  %
+%  Upper bound on the autocorrelation parameters for maximum likelihood
+%  estimation.
+%  %
+%  \default{None}
+  \item \xmlNode{normalize\_y}, \xmlDesc{boolean, optional field}, if True, the
+  observations $y$ are centered and reduced w.r.t. means and
   standard deviations estimated from the n\_samples observations provided.
   %
   \default{True}
-  \item \xmlNode{nugget}, \xmlDesc{float, optional field},specifies a nugget
-  effect to allow smooth predictions from noisy data.
-  %
-  The nugget is added to the diagonal of the assumed training covariance.
-  %
-  In this way it acts as a Tikhonov regularization in the problem.
-  %
-  In the special case of the squared exponential correlation function, the
-  nugget mathematically represents the variance of the input values.
-  %
-  \default{10 * MACHINE\_EPSILON}
+%  \item \xmlNode{nugget}, \xmlDesc{float, optional field},specifies a nugget
+%  effect to allow smooth predictions from noisy data.
+%  %
+%  The nugget is added to the diagonal of the assumed training covariance.
+%  %
+%  In this way it acts as a Tikhonov regularization in the problem.
+%  %
+%  In the special case of the squared exponential correlation function, the
+%  nugget mathematically represents the variance of the input values.
+%  %
+%  \default{10 * MACHINE\_EPSILON}
```

Comment:
commented lines? Do you still want to keep them?

Reply:
removed